_commontech/general_product_overview.md

```diff
@@ -76,4 +76,4 @@ Hybrid Block Storage nodes provide lower-cost and higher-density Block Storage.
 	• You can select Block Storage: SSD or HYBRID
 	• You can select Object Storage 
 
-Your purchase of IBM Blue Box Dedicated Cloud includes 24/7 technical support. The Box Panel product provides an interface for opening a support ticket. You also can call Blue Box Support at 1-800-613-4305 or email us at *support@bluebox.net*.
+Your purchase of IBM Bluemix Dedicated on OpenStack includes 24/7 technical support. The Box Panel product provides an interface for opening a support ticket. You also can call Bluemix on OpenStack Support at 1-800-613-4305 or email us at *support@bluebox.net*.
```

Comment:
Actually, Dedicated and Local BOTH come with 24/7 technical support. Here's some new wording:

Your purchase of IBM Bluemix on OpenStack, Dedicated or Local, includes 24/7 technical support.